src/test/java/com/genability/client/api/service/TimeOfUseServiceTests.java

```diff
@@ -213,7 +213,17 @@ public void deletePrivateTouGroupWorksCorrectly() {
 	}
 
 	private void cleanUpPrivateTou(TimeOfUseGroup grp) {
-		touService.deletePrivateTimeOfUseGroup(grp.getLseId(), grp.getTouGroupId());
+		try {
+			// Check if the TOU group has been deleted
```

Comment:
Add context to someone reading the code without knowledge of this issue: "Try to delete the private TOU only if it still exists (sometimes it might be already deleted)"